src/schema/core.yaml

```diff
@@ -1592,6 +1593,7 @@ slots:
     inlined_as_list: true
 
   known_as:
+    description: An identifier of a ChemicalEntity.
```

Comment:
It would be helpful for me to have a more detailed description here; in my (albeit strange) brain, "known_as" typically refers to a "name" or "alias" vs. an identifier...this could be tangential to this PR though!  happy to refine description that in a subsequent PR.

Reply:
I agree.  I find the name of this slot unintuitive and frankly problematic for the reason you point out.  It was introduced during the Berkeley refactoring and there are no data in mongo using this slot, so if you can help to think of a more appropriate slot name and description, that would be much appreciated!

Reply:
The only domain of the slot `known_as` is the class [`PortionOfSubstance`](https://microbiomedata.github.io/berkeley-schema-fy24/PortionOfSubstance/).  I'd advocate to make a better slot name and description before we use it (aka, now ;) ).